check/action_retrier.go

```diff
@@ -0,0 +1,24 @@
+package check
+
+import (
+	"gopkg.in/eapache/go-resiliency.v1/retrier"
+	"time"
+)
+
+type ActionRetrier struct {
+	*retrier.Retrier
+}
+
+type ActionRetrierConfig struct {
+	NumOfRetries uint8
+	Amount       time.Duration
+}
+
+func NewActionRetrier(cfg *ActionRetrierConfig) ActionRetrier {
```

Comment:
why are we passing by reference here? The `NewActionRetrier` is not (and should not) modify the configuration